x/evm/keeper/abci.go

```diff
@@ -44,5 +52,33 @@ func (k *Keeper) EndBlock(ctx context.Context) error {
 	bloom := ethtypes.BytesToBloom(k.GetBlockBloomTransient(infCtx).Bytes())
 	k.EmitBlockBloomEvent(infCtx, bloom)
 
+	feeCollectorAddr := authtypes.NewModuleAddress(authtypes.FeeCollectorName)
```

Comment:
Let's extract the new logic to a private helper function.

Reply:
[f826379](https://github.com/mezo-org/mezod/pull/417/commits/f826379e7434f503f9e6e231bcd32f55cce0864b)